File: python/test/test_fill.py
import ThrustRTC as trtc


darr = trtc.device_vector('int32_t', 5)
trtc.Fill(darr, trtc.DVInt32(123))
print (darr.to_host())

trtc.Fill(darr.range(1,3), trtc.DVInt32(456))
print (darr.to_host())

File: java/test/test_fill.java
import java.util.*;
import JThrustRTC.*;

public class test_fill
{
	public static void main(String[] args) 
	{
		DVVector vec_to_fill = new DVVector("int32_t", 5);
		TRTC.Fill(vec_to_fill, new DVInt32(123));
		System.out.println(Arrays.toString((int[])vec_to_fill.to_host()));
	}	

}
